plugins/parodos/src/components/workflow/workflowDetail/topology/useDemoPipelineNodes.tsx

```diff
@@ -19,8 +19,9 @@ export function useDemoPipelineNodes(
   workflowTasks: WorkFlowTask[],
 ): PipelineNodeModel[] {
   const getStatus: any = (status: string) => {
```

Comment:
```suggestion
  const getStatus = (status: WorkflowTask['status') => {
    if (status === 'COMPLETED') return RunStatus.Succeeded;
    else if (status === 'IN_PROGRESS')
      return RunStatus.InProgress;
```

Reply:
I've not tried this code out but you we can have a  strongly typed `status` type.

Is it lower case coming from the API?

Reply:
the status enum from backend will be uppercase